### PR TITLE
move betas to 'get involved'

### LINKED ISSUE #258: Confusing link to android client
In https://nextcloud.com/install/#install-clients, under mobile apps, it looks like the apps are in `beta` state, while I've been told in IRC (#nextcloud-dev, by nickvergessen) that it's actually 2 distinct links, to the stable version and to the beta version. This isn't clear from the UI.

### PR DESCRIPTION
**Pull request by jospoortvliet**
Should fix #258

### CODE REVIEW

**Comment by MariusBluem** [comment]
I think Google Play is RC and F-Droid is Beta @AndyScherzinger 🤔

**Comment by AndyScherzinger** [comment]
@MariusBluem exactly, while both also ship the stable of course.

**Comment by MariusBluem** [comment]
> while both also ship the stable of course.

Of course, but this does not apply to the **Get involved** section 😜😅

**Comment by MariusBluem** [comment]
Just changed the distinction of RC and BETA channel and corrected F-droid to F-Droid ;)
Please review! @AndyScherzinger @nickvergessen @jospoortvliet

File: page-install.php [hunk 1]
```diff
@@ -81,8 +81,9 @@
...
 		<p><a href="<?php echo $DOWNLOAD_SERVER_DAILY_TAR; ?>" class="btn btn-lg btn-default"><i class="icon-archive"></i>  <?php echo $l->t('Daily build');?></a></p>
+		<p><?php echo $l->t('Help test our clients:');?> <a target="_blank" href="<?php echo $DOWNLOAD_CLIENT_MOBILE_ANDROID_BETA; ?>"><?php echo $l->t('Android Release Candidate client on Play store');?></a> and <a target="_blank" href="<?php echo $DOWNLOAD_CLIENT_MOBILE_FDROID_BETA; ?>"><?php echo $l->t('Android Beta client on F-Droid');?></a></p>
 		<?php if(!empty($SERVER_TESTING_VERSION)) { ?>
-			    <p><?php echo $l->t('Latest testing version');?>: <span class="label label-blue"><?php echo $SERVER_TESTING_VERSION; ?></span></p>
+		<p><?php echo $l->t('Latest testing version');?>: <span class="label label-blue"><?php echo $SERVER_TESTING_VERSION; ?></span></p>
```
Comment (nickvergessen) [inline page-install.php]: Once the colon is translated and once not.
/me cries for unification

**Pull request merged**